LayoutTests/http/tests/media/fairplay/eme2016.js

```diff
@@ -0,0 +1,105 @@
+async function fetchBuffer(url) {
+    try {
+        let result = await fetch(url);
+        let buffer = await result.arrayBuffer();
+        consoleWrite(`FETCH: ${url} OK`);
+        return buffer;
+    } catch(e) {
+        consoleWrite(`FETCH: ${url} ERROR`);
+        failTest();
+    }
+}
+
+async function getResponse(event) {
+    var licenseResponse = await fetch('resources/index.py', {
+        method: 'POST',
+        headers: new Headers({'Content-type': 'application/x-www-form-urlencoded'}),
+        body: JSON.stringify({
+            "fairplay-streaming-request" : {
+                "version" : 1,
+                "streaming-keys" : [{
+                    "id" : 1,
+                    "uri" : 'skd://twelve',
+                    "spc" : base64EncodeUint8Array(new Uint8Array(event.message)),
+                }]
+            }
+        }),
+    });
+    consoleWrite('PROMISE: licenseResponse resolved');
+    var license = await licenseResponse.text();
+
+    responseObject = JSON.parse(license.trim());
```

Comment:
Either `response.json()` can be used instead of the `.text` `.trim` and `JSON.parse`
OR
based off the test failure and if it's feasible that the server fails, it might make sense to try/catch the JSON parse and dump the text if it fails, rather than having the cryptic `SyntaxError: JSON Parse error: Unrecognized token` error.

Reply:
Okay, I'll add a try/catch block around this, and just re-use this method in all the other .html tests (rather than make the same change in each place).